en/developer_guide/multi_device.rst

```diff
@@ -6,7 +6,7 @@ Prerequisite
 
 To enable the multi-device rendering, we assume that the server entry point (webserver or reverse-proxy)
  implements the Wurfl librairy and is able to add a request header with the device type.
```

Comment:
Is an implementation of wurfl trully required ? Open Orchestra only needs to get a request tagged with xua-device, and a twig (or other language) template matching that tag.
With this point of view, wurfl is not a prerequisite, only something to tag the client request.